UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/appsettings.json

```diff
@@ -36,26 +36,30 @@
         "Start": "0"
     },
     "RadioNavigationalWarningsContext": {
-        "ConnectionString": ""
+      "ConnectionString": ""
     },
     "AzureAdB2C": {
-        "Instance": "",
-        "ClientId": "",
-        "ClientSecret": "",
-        "Domain": "",
-        "SignUpSignInPolicyId": "B2C_1A_SignIn_SPA",
-        "CallbackPath": "/signin-oidc",
-        "SignedOutCallbackPath": "/signout-callback-oidc",
-        "RedirectBaseUrl": "",
-        "Scope": ""
+      "Instance": "",
+      "ClientId": "",
+      "ClientSecret": "",
+      "Domain": "",
+      "SignUpSignInPolicyId": "B2C_1A_SignIn_SPA",
+      "CallbackPath": "/signin-oidc",
+      "SignedOutCallbackPath": "/signout-callback-oidc",
+      "RedirectBaseUrl": "",
+      "Scope": ""
     },
     "CacheConfiguration": {
-        "CacheStorageAccountName": "",
-        "CacheStorageAccountKey": "",
-        "FssWeeklyBatchSearchTableName": "FssWeeklyBatchSearchResponseCache",
-        "FssCacheResponseTableName": "FssCacheResponseTable",
-        "IsFssCacheEnabled": true,
-        "CacheTimeOutInMins": 30
+      "CacheStorageAccountName": "",
+      "CacheStorageAccountKey": "",
+      "FssWeeklyBatchSearchTableName": "FssWeeklyBatchSearchResponseCache",
+      "FssCacheResponseTableName": "FssCacheResponseTable",
+      "MsiBannerNotificationTableName": "MsiBannerNotificationTable",
```

Comment:
Should this not be in the BannerNotificationConfiguration ?

If it's for a technical reason (say, that passing this value to the helpers from the other object would complicate data flow), that would be okay.

Reply:
Moved the MsiBannerNotificationTableName to the cache configuration.